charts/umbrella/values.yaml

```diff
@@ -681,6 +681,149 @@ managed-identity-wallet:
       nginx.ingress.kubernetes.io/cors-allow-methods: GET
       nginx.ingress.kubernetes.io/enable-cors: "true"
 
+bpdm:
```

Comment:
bpdm-gate, bpdm-pool and bpdm-orchestrator expect default permissions that are different from the Central-IDP version in the umbrella chart. Therefore the default permissions have to be overwritten like seen here #108